.travis.yml

```diff
@@ -2,6 +2,24 @@ language: r
 cache: packages
 sudo: false
 warnings_are_errors: false
+r_packages:
+	- microbenchmark
```

Comment:
These lines (ones with red markings) are causing the travis check to fail. Mostly due to a tab/space issue.
Just retype them in, should correct the error.